### PR TITLE
Added ui components tab toggling

### PR DESCRIPTION
**Pull request by bohdan-harniuk**
**Description** (*)
Improved UX for entity manager form. Added UI components tab toggling depending on Create Admin UI Components checkbox value.

ezgif com-gif-maker (2)

Improving was suggested by @eduard13 in the https://github.com/magento/magento2-phpstorm-plugin/pull/483

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by eduard13 — CHANGES_REQUESTED** [review]
Thank you for adding this improvement. But the "Admin Ui Components" tab looks a bit strange now:
image

On your screenshot might also be seen, that has a rectangle around the form. Could you please check this issue?

Thank you.

File: src/com/magento/idea/magento2plugin/actions/generation/dialog/NewEntityDialog.form [hunk 1]
```diff
@@ -137,12 +143,19 @@
...
           </grid>
-          <grid id="e3309" layout-manager="GridLayoutManager" row-count="8" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+          <grid id="e3309" binding="uiComponentsPanel" layout-manager="GridLayoutManager" row-count="8" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
             <margin top="10" left="10" bottom="0" right="10"/>
             <constraints>
               <tabbedpane title="Admin UI Components"/>
             </constraints>
-            <properties/>
+            <properties>
+              <background color="-12828863"/>
```
Comment (eduard13) [inline src/com/magento/idea/magento2plugin/actions/generation/dialog/NewEntityDialog.form:152]: I think this is the reason of having that style issue

Comment (bohdan-harniuk) [reply src/com/magento/idea/magento2plugin/actions/generation/dialog/NewEntityDialog.form:152]: Thank you! I'll take a look.

**Comment by bohdan-harniuk** [comment]
Hi, @eduard13
I've removed external changes, but unfortunately, I cannot change Dracula theme to the light (only redactor area changes its colour)...could you please take a look if everything is ok?

Screenshot 2021-02-12 at 12 24 13

**Review by eduard13 — APPROVED** [review]
Good job 👍

**Pull request merged**